lib/src/dbs/options.rs

```diff
@@ -11,16 +14,18 @@ use std::sync::Arc;
 /// whether field/event/table queries should be processed (useful
 /// when importing data, where these queries might fail).
 
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug)]
 pub struct Options {
+	/// Current Node ID
+	pub id: Uuid,
 	/// Currently selected NS
 	pub ns: Option<Arc<str>>,
 	/// Currently selected DB
 	pub db: Option<Arc<str>>,
 	/// Connection authentication data
 	pub auth: Arc<Auth>,
 	/// Approximately how large is the current call stack?
-	dive: u8,
+	pub dive: u8,
```

Comment:
I mentioned this in an earlier PR review. Is it being reverted? (intentionally?)

Reply:
Ah no, I didnt reconcile. Thanks!